contrib/gitian-descriptors/gitian-linux.yml

```diff
@@ -35,7 +35,7 @@ script: |
 
   WRAP_DIR=$HOME/wrapped
   HOSTS="i686-pc-linux-gnu x86_64-linux-gnu arm-linux-gnueabihf aarch64-linux-gnu"
-  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --disable-gui-tests"
+  CONFIGFLAGS="--enable-glibc-back-compat --enable-reduce-exports --disable-bench --disable-gui-tests --disable-wallet --without-gui --without-miniupnpc --enable-mini"
```

Comment:
can we create a seperate gitian file for the linux mini build? I believe it can be almost identical except for the line above. Reason being; there will be people that want to run the full linux version as well.

Reply:
ok, I will add new file gitian-linux-mini.yml